README.md

```diff
@@ -1,4 +1,331 @@
-# Backplane-CLI
+# backplane-cli
 
-## What it does
-Backplane-cli is a CLI tool to interact with Backplane-api, which is a management service that consolidates all access to OSD and ROSA clusters and cloud provider resources. Backplane handles authentication and authorization through OCM profiles, practices least privilege access to those OSD/ROSA clusters and cloud resources, and provides auditing capabilities for compliance and security.
+backplane-cli is a CLI tool to interact with backplane api.
+
+The onboarding documentation can be found on [The Source Wiki](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_user_documentation).
+
+## Install
+
+### Option 1: Use `go install`
+
+You need to have `go` installed, if you want to use this method,
+the minimal version required is `go 1.16`. You can check this with `go version`
```

Comment:
Should be go 1.18 ? https://github.com/openshift/backplane-cli/blob/main/go.mod#L3